practitioner-service/src/test/java/uk/ac/ox/ndph/mts/practitioner_service/TestAuthorisationConfigurationProvider.java

```diff
@@ -20,9 +20,8 @@ public AuthorisationService authorisationService() {
         var mockService = Mockito.mock(AuthorisationService.class);
         Mockito.when(mockService.authorise(anyString())).thenReturn(true);
         Mockito.when(mockService.authorise(anyString(), anyString())).thenReturn(true);
-        Mockito.when(mockService.authorise(anyString(), anyList(), anyString())).thenReturn(true);
         Mockito.when(mockService.authorise(anyString(), anyList())).thenReturn(true);
-        Mockito.when(mockService.authoriseUserRoles(anyString())).thenReturn(true);
+        Mockito.when(mockService.authUserRoles(anyString())).thenReturn(true);
```

Comment:
general tidy as I wanted to make the methods less verbose to allow checkstyle to pass for line length